Pod/Classes/VOKManagedObjectMap.h

```diff
@@ -84,4 +84,18 @@
  */
 + (NSArray *)mapsFromDictionary:(NSDictionary *)mapDict;
 
+/**
+ Default formatter used for date fields. This is the RFC 3339 format, with
+ milliseconds included.
+ @return            Default date formatter
+ */
++ (NSDateFormatter *)vok_defaultDateFormatter;
+
+/**
+ Default formatter used for date fields. This is the RFC 3339 format, just like
+ the one returned by vok_defaultDateFormatter, but with milliseconds ommitted.
```

Comment:
Micro. I realized halfway through that I had been using the wrong prefix. Fixing now…